### PR TITLE
refactor: refactor action-runner-scheduler to pb provider

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
refactor action-runner-scheduler to pb provider

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda-org.erda.cloud/erda/dop/projects/387/issues/all?id=316432&issueFilter__urlQuery=eyJzdGF0ZXMiOls0NDAyLDcxMDQsNzEwNSw0NDAzLDQ0MDQsNzEwNiw0NDA2LDQ0MDcsNDQxMiw0NTM4LDQ0MTMsNDQxNCw0NDE1LDQ0MTZdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&iterationID=1250&type=TASK)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  refactor action-runner-scheduler to pb provider            |
| 🇨🇳 中文    |    action-runner-scheduler重构为provider并入pipeline          |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: api/proto/core/pipeline/action_runner_scheduler/runner_task.proto [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  string taskID = 2 [json_name = "task_id"];
+  string status = 3;
+  string contextDataUrl = 4 [json_name = "context_data_url"];
+  string resultDataUrl = 5 [json_name = "result_data_url"];
+}
+
+message RunnerTaskUpdateResponse {}
+
+message LogCollectRequest {
+  string source = 1;
```
Comment (sfwn) [inline api/proto/core/pipeline/action_runner_scheduler/runner_task.proto:142]: use https://github.com/googleapis/googleapis/blob/master/google/api/http.proto#L352

File: api/proto/core/pipeline/action_runner_scheduler/runner_task.proto [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+message LogCollectResponse {}
```
Comment (sfwn) [inline api/proto/core/pipeline/action_runner_scheduler/runner_task.proto:145]: use https://github.com/googleapis/googleapis/blob/master/google/api/http.proto#L360

**Review by sfwn — CHANGES_REQUESTED** [review]

File: api/proto/core/pipeline/action_runner_scheduler/runner_task.proto (hunk 1 above)
Comment (chengjoey) [reply api/proto/core/pipeline/action_runner_scheduler/runner_task.proto:136]: adjusted, fill request body data to content field

**Review by sfwn — DISMISSED** [review]

**Pull request merged**